### ISSUE TITLE
[INFO] act_sensor_time can also be the timestamp of meter installation

### ISSUE THREAD

**Issue by rs38** [spacemanspiff2007/SmlLib#16]
info only:
in my case it's showing the "time" (=seconds) since the smartmeter was installed and activated.
act_sensor_time / SML_timestamp

**Comment by spacemanspiff2007** [comment]
I am sorry I don't understand the issue.
Does it not resolve the timestamp correctly as an int?

**Comment by rs38** [comment]
that's why I said "info only".
I saw some time fiddling code where the author commented like he was unsure. 

```
msg = m.format_msg()
        for line in msg.splitlines():
            if 'act_sensor_time' in line:
                seconds = int( line.split(':')[1].strip())
                return datetime.datetime.fromtimestamp(seconds+1673770017)
```

1673770017 = Unix time of meter install date

to be honest I was looking for a timestamp propertiy in the OBIS part and could not find it.

**Comment by rs38** [comment]
it's a timestamp of time **_since_** installation.
similar to the example data you provide in the readme:
`act_sensor_time : 226361515`
which also cannot mean true unix timestamp but like 7 years ago.

**Comment by spacemanspiff2007** [comment]
`actSensorTime` is the name in the spec so I named the attribute `act_sensor_time`

**Comment by spacemanspiff2007** [comment]
> not sure if you got my point :)

I'm not sure, either. Can you rephrase your point.

**Comment by rs38** [comment]
I have no idea, how to express it better....
this?
`[INFO] act_sensor_time can also be the timespan in seconds since meter installation`

**Comment by spacemanspiff2007** [comment]
Do you think I should add a generel field info section where I put this?
As more of a description what the fields could mean?
